README.md

```diff
@@ -145,6 +145,18 @@ func case5(x int) {                                //-|
 | —format | Format of the diff coverage report, one of: html, json, markdown |
 | —excludes | Exclude files for diff coverage inspection |
 
+## Miscellaneous
+
+### About `_test.go`
+
+`gocover` highly relies on cover profiles generated by `go test`. `go test` generates cover profiles for a specific package at circumstance that there is at least one `_test.go` in the package.
+
+So there would be two situations:
+1. A new package has `_test.go` file. `go test` can generate cover profile demandly, then `gocover` will generate diff coverage for the package.
```

Comment:
this issue doesn't only apply to "new" package, right?

Reply:
Yeah. It can happen in an existing package that does not have any _test.go file.